CHANGELOG.md

```diff
@@ -62,6 +62,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Fixed multiple early stopping callbacks ([#6197](https://github.com/PyTorchLightning/pytorch-lightning/pull/6197))
 
+- Fixed a bug regarding logger on TPU ([#6221](https://github.com/PyTorchLightning/pytorch-lightning/pull/6221))
```

Comment:
How about making it a bit more specific?
```suggestion
- Fixed `AttributeError` when `logger=None` on TPU ([#6221](https://github.com/PyTorchLightning/pytorch-lightning/pull/6221))
```

Reply:
Thanks, good.

> How about making it a bit more specific?

Thanks, good.